content/en/docs/reference/access-authn-authz/service-accounts-admin.md

```diff
@@ -62,10 +62,9 @@ It acts synchronously to modify pods as they are created or updated. When this p
 
 #### Bound Service Account Token Volume
 
-{{< feature-state for_k8s_version="v1.21" state="beta" >}}
+{{< feature-state for_k8s_version="v1.22" state="stable" >}}
 
-When the `BoundServiceAccountTokenVolume` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) is enabled, the service account admission controller will
-add the following projected volume instead of a Secret-based volume for the non-expiring service account token created by Token Controller.
+The service account admission controller will add the following projected volume instead of a Secret-based volume for the non-expiring service account token created by Token Controller.
```

Comment:
```suggestion
The ServiceAccount admission controller will add the following projected volume instead of a Secret-based volume for the non-expiring service account token created by Token Controller.
```
The [admission controller page](https://kubernetes.io/docs/reference/access-authn-authz/admission-controllers/#serviceaccount) uses camel case for `ServiceAccount`